app/src/main/java/com/kabouzeid/gramophone/ui/fragments/player/card/CardPlayerFragment.java

```diff
@@ -244,8 +258,15 @@ public boolean onMenuItemClick(MenuItem item) {
     }
 
     private void setUpRecyclerView() {
+        recyclerViewTouchActionGuardManager = new RecyclerViewTouchActionGuardManager();
```

Comment:
A lot of these changes appear to be the same between the different player fragments. Could you move the common logic into the parent class?

Reply:
Done.